ticketutil/jira.py

```diff
@@ -312,6 +312,17 @@ def change_status(self, status):
         params = {'transition': {}}
         params['transition']['id'] = status_id
 
+        # Some of the ticket fields need to be in a specific form for the tool to be Resolved from In Progress/To Do/Groomed
+        fields = _prepare_ticket_fields(kwargs)
+
+        # Parameters for the ticket to be Resolved from To Do/Groomed
```

Comment:
Remove this line because it's specific to our RH Jira instance.

Reply:
The changes have been incorporated in the latest commit